updating/updating-cluster-cli.adoc

```diff
@@ -18,6 +18,11 @@ You can update, or upgrade, an {product-title} cluster within a minor version by
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
++
+[IMPORTANT]
+====
+When an update is blocked, the Cluster Version Operator (CVO) reports the status of any blocked components while attempting to reconcile the desired update. Rolling your cluster back to a previous version is not supported, therefore etcd backups are an important recovery option. If your update is blocked, contact Red Hat support.
```

Comment:
I'd like to avoid using the words "an update is blocked" just to ensure we're not confusing people with blocked edges. Possible alternatives
"When an update does not complete successfully.." 
"When an update fails.."
"When upgrade completion is blocked.."

I'd also like to say "disaster recovery" rather than just "recovery" so that we set the bar very high, 99% of the time it's a better option to fix whatever problems they've encountered unless their cluster is completely taken offline due to an upgrade.

Reply:
Perhaps "When an upgrade results in cluster failure..."